src/PermanentFilterProvider.tsx

```diff
@@ -2,34 +2,61 @@ import { noop } from 'lodash';
 import React, { createContext, useCallback, useEffect, useReducer } from 'react';
 
 import { readFromStore } from './helpers';
-import { DATA_PROVIDER_FILTER_KEY, permanentFilterReducer } from './reducers';
+import {
+  DATA_PROVIDER_FILTER_KEY,
+  MOWAS_REGIONAL_KEYS,
+  MowasFilterAction,
+  MowasFilterReducerAction,
+  mowasRegionalKeysReducer,
+  permanentFilterReducer
+} from './reducers';
 import { FilterAction, FilterReducerAction } from './types';
 
 type PermanentFilterProviderValues = {
-  state: string[];
-  dispatch: React.Dispatch<FilterReducerAction>;
+  dataProviderState: string[];
+  dataProviderDispatch: React.Dispatch<FilterReducerAction>;
+  mowasRegionalKeysState: string[];
+  mowasRegionalKeysDispatch: React.Dispatch<MowasFilterReducerAction>;
 };
 
 export const PermanentFilterContext = createContext<PermanentFilterProviderValues>({
-  dispatch: noop,
-  state: []
+  dataProviderState: [],
+  dataProviderDispatch: noop,
+  mowasRegionalKeysState: [],
+  mowasRegionalKeysDispatch: noop
```

Comment:
alphabetical order would be dispatch before state, right?

Reply:
Yes, you are right. I can alphabetise here. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1063/commits/9114e12a4846fe105183b67e08910216b6e8e4ee